universal-login-contracts/contracts/IdentityMasterCopy.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.5.2;
+
+import "./common/MasterCopy.sol";
+import "./Identity.sol";
+
+
+contract IdentityMasterCopy is MasterCopy, Identity {
+    constructor(bytes32 _key, bytes32 _hashLabel, string memory _name, bytes32 _node, ENS ens, FIFSRegistrar registrar, PublicResolver resolver)
+        Identity(_key, _hashLabel, _name, _node, ens, registrar, resolver) public {
+    }
```

Comment:
Identity should not inherit from ENSRegistred anymore.

Reply:
#someotherpr 
FIFSRegistrar registrar -> can be obtained from ENS
PublicResolver resolver -> can be obtained from ENS